src/hooks/audio/useAudioPlayer.tsx

```diff
@@ -10,17 +10,20 @@ import { useEventListener, useMKEventListener } from 'hooks';
 import * as ConversionUtils from 'utils/conversion';
 
 import { useMusicKit, useSettings, useSpotifySDK, VOLUME_KEY } from '../';
+import { IpodEvent } from 'utils/events';
+
+const defaultPlatbackInfoState = {
```

Comment:
To DRY things up a bit: we can define the default state here, use it to infer the types, and pass it into the `useState` as the initial value